packages/vm/lib/runTx.ts

```diff
@@ -186,7 +184,7 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
    */
   if (results.execResult.selfdestruct) {
     const keys = Object.keys(results.execResult.selfdestruct)
-    for (const k of keys) {
+    for await (const k of keys) {
```

Comment:
Just for understanding: what is this double-await changing here?

Reply:
@holgerd77 `for await` is for async iterators, but Object.keys does not return an async iterator, so this is a mistake

Reply:
@ryanio nit ^

Reply:
@holgerd77 @kumavis noted, thanks, I will address in a follow-up PR.

Reply:
@kumavis thanks for having another look here! 🙂